scripts/preprocess_run.py

```diff
@@ -17,12 +18,19 @@
 path = os.path.join(DATA_RAW, 'countries.csv')
 pop_tif_loc = os.path.join(DATA_RAW, 'WorldPop', 'ppp_2020_1km_Aggregated.tif')
 
-country = ProcessCountry(path, 'USA')
-national_outline = country.process_country_shapes()
+#country = ProcessCountry(path, 'KEN')
```

Comment:
Again, try to keep it clean by removing unhashed code. If you don't want to delete, you can always push it to github, then delete it and repush, and it should be in the history for later if necessary.